docs/software/sciapps/lammps.md

```diff
@@ -1,4 +1,328 @@
+It can model atomic, polymeric, biological, metallic, granular, and coarse-grained systems using a variety of force fields and boundary conditions. 
+The current version of LAMMPS is written in C++.
+
+!!! note "uenvs"
+    [LAMMPS](https://www.lammps.org/) is provided on [ALPS][platforms-on-alps] via [uenv][ref-uenv].
+    Please have a look at the [uenv documentation][ref-uenv] for more information about uenvs and how to use them.
+
+
+??? note "Licensing terms and conditions"
+    [LAMMPS] is a freely-available open-source code, distributed under the terms of the [GNU Public License](http://www.gnu.org/copyleft/gpl.html).
+
+## Running LAMMPS
+
+### Loading LAMMPS Interactively
+
+On Alps, LAMMPS is precompiled and available in a [uenv][ref-uenv]. 
+LAMMPS has been built with the [Kokkos](https://docs.lammps.org/Speed_kokkos.html) and GPU packages separately.
+
+To find which LAMMPS uenv is provided, you can use the following command:
+
+```
+uenv image find lammps
+```
+
+which will list several available LAMMPS uenv images.
+We recommend that you regularly check for the latest version.
+Please see the documentation here for further details: https://eth-cscs.github.io/cscs-docs/software/uenv/#finding-uenv.
+
+To obtain this image, please run:
+
+```bash
+uenv image pull lammps/2024:v2
+```
+
+To start the uenv for this specific version of LAMMPS, you can use:
+
+```bash
+uenv start --view kokkos lammps/2024:v2
+```
+
+You can load the `kokkos` or `gpu` view from the uenv to make the `lmp` executable available.
+The executable in both these views support GPUs:
+
+=== "Kokkos"
+    ```bash
+    #lammps +kokkos package
+    uenv start --view kokkos lammps/2024:v2
+    ```
+=== "GPU"
+    ```bash
+    #lammps +gpu package
+    uenv start --view gpu lammps/2024:v2
+    ```
+
+A development view is also provided, which contains all libraries and command-line tools necessary to build LAMMPS from source, without including the LAMMPS executable:
+
+=== "Kokkos"
+    ```bash
+    # build environment for lammps +kokkos package, without providing lmp executable
+    uenv start --view develop-kokkos lammps/2024:v2
+    ```
+=== "GPU"
+    ```bash
+    # build environment for lammps +gpu package, without providing lmp executable
+    uenv start --view develop-gpu lammps/2024:v2
+    ```
+
+### Running LAMMPS with Kokkos on the HPC Platform
+
+To start  a job, two bash scripts are potentially required: a [SLURM] submission script, and a wrapper for `numactl` which sets up CPU and memory binding.
+
+The submission script is the following:
+
+```bash title="run_lammps_kokkos.sh"
+#!/bin/bash -l
+#SBATCH --job-name=<JOB_NAME>
+#SBATCH --time=01:00:00 (1)
+#SBATCH --nodes=2                                                                        
+#SBATCH --ntasks-per-node=4  (2)
+#SBATCH --gpus-per-node=4
+#SBATCH --gpus-per-task=1
+#SBATCH --gpu-bind=per_task:1
```

Comment:
I think the `--gpus-per-task=1` alone covers this. I've never seen the `--gpus-per-node=4` + `--gpu-bind=per_task:1` form before so can't say for sure if that does something different, but if the goal is to have four ranks and one GPU per task, then in my experience just having `--gpus-per-task=1` is sufficient.

```suggestion
#SBATCH --gpus-per-task=1
```

If you're unsure and would like to leave the other options there for now that's also ok by me. @RMeli?

Reply:
I've also never seen this combination. If they are equivalent, I'd go for `--gpus-per-task=1` for consistency in our documentation. If it does something different, maybe it is worth commenting/adding a note explaining the difference?